frontend/src/components/uam/uam.tsx

```diff
@@ -18,10 +18,14 @@ const UAM: FC = () => {
   const columns: Column<UsersGetAllItemResponseDto>[] = getColumns();
   const rows: UsersGetAllItemResponseDto[] = getRows(users);
 
+  const deleteUser = (userId: string): void => {
```

Comment:
```suggestion
  const handleUserDelete = (userId: string): void => {
```
`handle` for component's cbs